### PR TITLE
Remove smart contract duplicated content

### PR DESCRIPTION
**Pull request by jimmychu0807**
The original content in `index.md` is already covered in `ink-fundamentals.md` in more detailed form, so removing the dup content.

### CODE REVIEW

**Comment by NukeManDan** [comment]
https://wiki.polkadot.network/docs/en/build-build-with-polkadot
https://wiki.polkadot.network/docs/en/build-smart-contracts

(I have a pending PR to tweak a bit, we will see if it's approved) https://github.com/w3f/polkadot-wiki/pull/2076

These pages also detail some/most of what we have here. Should this be cross-linked? Or might we want to reduce duplication, and have the wiki point here or vice versa?

File: docs/knowledgebase/smart-contracts/index.md [hunk 1]
```diff
@@ -1,31 +1,190 @@
...
+
+**In summary**, Substrate Smart Contracts:
+
+- Are inherently safer to the network.
+- Have built in economic incentives against abuse.
+- Have computational overhead to support graceful failures in logic.
+- Have a lower bar to entry for development.
+- Enable fast pace community interaction through a playground to write new logic.
+
+### Runtime Modules
```
Comment (NukeManDan) [inline docs/knowledgebase/smart-contracts/index.md:79]: should we call them pallets?

Comment (jimmychu0807) [reply docs/knowledgebase/smart-contracts/index.md:79]: Right. The whole article is reviewed to replace all module to pallet.

**Comment by jimmychu0807** [comment]
@NukeManDan 
Thanks for the pointers to polkadot wiki. They are linked, and pointing out the usage of `module`. The article is written before we standardize to use `pallet`, so `module` has occurred at a few different places.

**Review by riusricardo — APPROVED** [review]

**Pull request merged**